src/test/shell/bazel/bazel_determinism_test.sh

```diff
@@ -72,6 +72,12 @@ function test_determinism()  {
     # Update the hash of bazel_tools in lockfile to avoid rerunning module resolution.
     new_hash=$(shasum -a 256 "src/MODULE.tools" | awk '{print $1}')
     sed -i.bak "/\"bazel_tools\":/s/\"[a-f0-9]*\"/\"$new_hash\"/" MODULE.bazel.lock
+    # TODO: Temporary hack for lockfile version mismatch, remove these lines after updating to 7.1.0
+    sed -i.bak 's/"lockFileVersion": 3/"lockFileVersion": 4/' MODULE.bazel.lock
+    sed -i.bak 's/^[[:space:]]*"name": "[^"]*~[^"]*",\?//' MODULE.bazel.lock
```

Comment:
You'll have to remove `\?`, it breaks this command on macOS. I guess it's also safe? Usually the name attribute is not the last one?

Reply:
OK, let me know if you want to me test the command.